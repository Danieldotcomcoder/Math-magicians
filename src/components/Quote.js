import React from 'react';

function Quote() {
  return (
    <div>
      <h1 className="quote">
        Mathematics is not about numbers, equations, computations, or algorithms:it is about
        understanding. - William Paul Thurston
      </h1>
    </div>

  );
}
export default Quote;
