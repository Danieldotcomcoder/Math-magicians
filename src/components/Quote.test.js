import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quote';

it('Renders Quote page', () => {
  const home = renderer.create(
    <Quote />,
  ).toJSON();
  expect(home).toMatchSnapshot();
});
