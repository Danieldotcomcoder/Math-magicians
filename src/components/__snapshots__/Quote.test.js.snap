// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Quote page 1`] = `
<div>
  <h1
    className="quote"
  >
    Mathematics is not about numbers, equations, computations, or algorithms:it is about understanding. -- William Paul Thurston
  </h1>
</div>
`;
